_harp/academy/_apply.md

```diff
@@ -9,11 +11,12 @@ We are now taking applications for the Autummn 2016 cohort (starting in October)
 
 We may also request a short in-person paired programming session (remote sessions can also be arranged).
 
-We offer places on a rolling basis. Early application is recommended.
+We offer places on throughout the months leading up to the application deadline. Early application is recommended so we know you are interested and can keep an eye out for you in our community.
```

Comment:
I thought 'rolling basis' wasn't clear to people who weren't familiar to our process.
Also thought we should explain why early application is a good thing.